Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -22,6 +22,17 @@ namespace DSP
 {
 namespace HLE
 {
+static bool HasLpf(u32 crc)
+{
+  switch (crc)
+  {
+  case 0x4e8a8b21:
+    return false;
+  default:
```

Comment:
I'm not convinced that the RS2 UCode is the only one to not have LPF support. I'll try to have a look through my research notes...